kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating Oracle DB service and RCU Schema for a Fusion Middleware domain
+
+This sample demonstrates how to create an Oracle DB service on Kubernetes cluster and create RCU schema on the Oracle DB being used by a Fusion Middleware domain.  
```

Comment:
on Kubernetes -> in a Kubernetes
and create -> and how to create
schema on the -> schema in the
being used by -> that will be used by

Reply:
Re-worded as follows ...
The sample scripts in this directory demonstrates how to
  *  start an Oracle DB service in a Kubernetes cluster
  *  stop an Oracle DB service in a Kubernetes cluster
  *  create RCU schema in the Oracle DB that will be used by a Fusion Middleware domain.
  *  delete RCU schema in the Oracle DB used by a Fusion Middleware domain.

Reply:
The sample scripts in this directory demonstrates how to:
    * Start an Oracle DB service in a Kubernetes cluster.
    * Stop an Oracle DB service in a Kubernetes cluster.
    * Create the RCU schema in the Oracle DB that will be used by a Fusion Middleware domain.
    * Delete the RCU schema in the Oracle DB used by a Fusion Middleware domain.